src/perl/Permabit/BlockDevice/VDO/LVMVDO.pm

```diff
@@ -443,4 +303,15 @@ sub enableDeduplication {
   }
 }
 
+########################################################################
+# Rename a VDO device. Currently LVM managed devices are the only
+# ones that support rename.
+#
+# @param newName The new name for the VDO device.
+##
+sub renameVDO {
+  my ($self, $newName) = assertNumArgs(2, @_);
+  confess("Failed to override the renameVDO method");
+}
+
```

Comment:
It feels weird to move this method for no obvious reason. Why not leave it where it was? (around line 197)

Reply:
This comment now applies to a lot more functions. For example, enable/disable ReadableStorage. I get that you need to change the methods, but putting them in a different place in the file makes the diff hard to read, and it's also hard to parse out what you really changed about those functions.

Basically, if a function existed in LVMManaged,pm, and it now exists in LVMVDO.pm, could you make sure the function order lines up so the diffs are clearer to read?
rename VDO() is an exmple of how I think this should be done. Make sure the function is in the same place in the file, so the diffs are dlear about what parts of the implementation changed and which didn't.

Reply:
I've changed the locations to hopefully be correct now. There are some functions that aren't there in the new file, but hopefully diff will deal.